tutorials/standalone-tensorflow-raspberry-pi/tf_server.py

```diff
@@ -0,0 +1,43 @@
+print '--- importing packages'
+
+from tensorflow.contrib import predictor
+import base64
+import sys
+import json
+import subprocess
+import datetime
+
+# change in the line below the directory where you hosted your model
```

Comment:
The first couple of lines of this comment block are a little awkward. A bit of reordering and some punctuation would help. Here's my attempt:

In the line below, specify the directory where you hosted your model.
This is the directory where the .pb file and variables directory are hosted.

Reply:
Done.